packages/core/ios/Fabric/LottieAnimationViewComponentView.h

```diff
@@ -0,0 +1,27 @@
+#ifdef __cplusplus
+#import <React/RCTViewComponentView.h>
+#import <react/renderer/components/lottiereactnative/Props.h>
+#import "LottieContainerView.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface LottieAnimationViewComponentView : RCTViewComponentView <LottieContainerViewDelegate>
+@end
+
+namespace facebook {
+namespace react {
+// In order to compare these structs we need to add the == operator for each
+bool operator==(const LottieAnimationViewColorFiltersStruct& a, const LottieAnimationViewColorFiltersStruct& b)
```

Comment:
https://github.com/reactwg/react-native-new-architecture/discussions/91#discussioncomment-4426469

Reply:
Please add this link as a reference inside the code and mark it as TODO or FIX.